app/experimenter/visualization/api/v3/views.py

```diff
@@ -111,12 +111,14 @@ def analysis_results_view(request, slug):
     }
 
     for window in windows:
-        data = raw_data[window] = JetstreamData(__root__=get_data(recipe_slug, window))
+        data = raw_data[window] = JetstreamData(
+            __root__=(get_data(recipe_slug, window) or [])
+        )
         result_metrics, primary_metrics_set, other_metrics = get_results_metrics_map(
             data, experiment.primary_outcomes, experiment.secondary_outcomes
         )
 
-        if data and window == "overall":
+        if len(data) > 0 and window == "overall":
```

Comment:
Ya, also fyi you can use [`__nonzero__`](https://docs.python.org/2/reference/datamodel.html#object.__nonzero__) to implement the bool check 👍

Reply:
Sorry sorry it's obviously [`__bool__`](https://docs.python.org/3/reference/datamodel.html#object.__bool__) in py3 XD my bad